public/app/features/alerting/unified/RuleEditor.test.tsx

```diff
@@ -294,7 +294,7 @@ describe('RuleEditor', () => {
             grafana_alert: {
               condition: 'B',
               data: getDefaultQueries(),
-              exec_err_state: 'Alerting',
+              exec_err_state: 'Error',
```

Comment:
We should use `GrafanaAlertStateDecision.Error` instead of 'Error' string

Reply:
I think I will leave this to be discussed within the frontend squad as I don't have a good understanding of the RuleEditor.